tests/datatypes/test_complex.py

```diff
@@ -8,6 +8,18 @@ def test_conjugate(self):
             print(x.conjugate())
             """)
 
+    def test_real(self):
+        self.assertCodeExecution("""
+            x = complex(1, 1)
+            print(x.real)
+            """)
+
+    def test_imag(self):
+        self.assertCodeExecution("""
+            x = complex(1, 2)
+            print(x.imag)
+            """)
```

Comment:
Could you merge these two tests into one?
They're pretty much related, and it makes the test case run slightly faster (each new test needs compiling and running code a few times).